project.clj

```diff
@@ -7,12 +7,21 @@
   :license {:name "Apache License, Version 2.0"
             :url "http://www.apache.org/licenses/LICENSE-2.0.html"}
 
-  :dependencies [[org.clojure/clojure "1.6.0"]
-                 [org.clojure/java.jdbc "0.3.2"]
-                 [org.postgresql/postgresql "9.3-1100-jdbc41"]
-                 [com.jolbox/bonecp "0.8.0.RELEASE" :exclusions [[org.slf4j/slf4j-api]]]
+  :dependencies [[com.jolbox/bonecp "0.8.0.RELEASE" :exclusions [[org.slf4j/slf4j-api]]]
+                 [com.zaxxer/HikariCP-java6 "2.3.7" :exclusions [[org.slf4j/slf4j-api]]]
+                 [metrics-clojure "2.5.1" :exclusions [[org.slf4j/slf4j-api]]]
```

Comment:
This is required because of how hikaricp wishes to register statistics, however - I'm tempted to leave this work or see how @cprice404's metrics work goes, since I presume we might need to somehow have an existing MetricRegistry passed to HikariCP. Anyway, lots of uncertainties here, we can have metrics without cprice's work, but it requires manual registration work for the data to get exposed via jmx:

```
(def JR (jmx/reporter {}))
(metrics.reporters.jmx/start JR)
```

These steps don't feel like they belong in jdbc-util obviously :-).

Right now, I'm using the default-registry provided out of metrics-clojure by default, so this should just work.